aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -468,6 +468,28 @@ Tensor sparse_coo_tensor(const Tensor& indices, const Tensor& values, IntArrayRe
       !options.has_layout() || options.layout() == kSparse,
       "expected sparse layout, but got layout ",
       options.layout());
+
+  if (indices.numel() > 0) {
+    Tensor min_indices =
```

Comment:
Is there a reason these indices need to be a Tensor? working with raw C++ numbers will be more performant, especially for just error checking.

Reply:
oh, sparse indices, i see.